modules/virt-autoupdate-custom-bootsource.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-autoupdate-custom-bootsource_{context}"]
+= Enabling automatic updates on custom boot sources
+
+{VirtProductName} automatically updates pre-defined boot sources by default. You must manually enable automatic imports and updates on any custom boot sources.
+
+.Procedure
+
+. Use the following command to enable automatic updates on a custom boot source:
++
+[source,terminal]
+----
+$ oc apply -n openshift-cnv -f add-custom-boot-sources.yaml
+----
+
++
+.Example in CentOS 7
+[source,yaml]
+----
+apiVersion: hco.kubevirt.io/v1beta1
+kind: HyperConverged
+metadata:
+  name: kubevirt-hyperconverged
+spec:
+  dataImportCronTemplates:
+  - metadata:
+      name: centos7-image-cron
+      annotations:
+        cdi.kubevirt.io/storage.bind.immediate.requested: "true" (1)
+    spec:
+      schedule: "0 */12 * * *" (2)
+      template:
+        spec:
+          source:
+            Registry: (3)
+              url: docker://quay.io/containerdisks/centos:7-2009
+          storage:
+            resources:
+              requests:
+                storage: 10Gi
+      managedDataSource: centos7 (4)
+      retentionPolicy: "None" (5)
+----
+<1> This annotation is required for storage classes with link:https://kubernetes.io/docs/concepts/storage/storage-classes/#volume-binding-mode[`volumeBindingMode`] set to `WaitForFirstConsumer`.
+<2> Schedule for the job specified in link:https://en.wikipedia.org/wiki/Cron[cron format].
+<3> Use to create a Data Volume from an registry source.
+<4> The image name must match the template name to use it as a boot source for the template.
```

Comment:
a UI user can:
a - just use the templae parameters default, then it's the default value of DATA_SOURCE_NAME
b - change the parameter, like any parameter, and then it will look for a diferent `DataSource` object that looks for a different image (assuming the dataSource exist) as a parameter a user can put anything.
c - override the "source" of dataVolumeTemplate  completely, then they can choose between url, registry and existing PVC, in this case the original `soureRef` or `source` are deleted and replaced with new source

Reply:
@yaacov Thanks for helping.
@sabrinajess Can you please change the line to something like
'<4> To let the image beeing detected as the default boot source for a template, the `managedDataSource` has to match the name of the `DataSource` in the related template.' ?

Reply:
Can do. Thanks for the clarity.